fastlane/lib/fastlane/actions/crashlytics.rb

```diff
@@ -28,11 +28,29 @@ def self.run(params)
         end
 
         UI.success('Uploading the build to Crashlytics Beta. Time for some ☕️.')
-        UI.verbose(command.join(" ")) if $verbose
-        Actions.sh(command.join(" "), log: false)
+
+        sanitizer = proc do |message|
+          message.gsub(params[:api_token], '[[API_TOKEN]]')
+                 .gsub(params[:build_secret], '[[BUILD_SECRET]]')
```

Comment:
I did not know that Ruby will read onto the next line in order to determine if a statement was complete.